GETTING_STARTED.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+    git clone https://github.com/black-parrot/pre-alpha-release.git
+    # make tools will clone, build and install toolchains needed for BlackParrot
+    # For faster builds, make tools -j is parallelizable!
+    make tools
+    # make tidy_tools will remove all tool build directories, while leaving the installation
+    # make progs will build all open source test programs that ship with BlackParrot
+    make progs
+    # make ucode will build the CCE microcode used for BlackParrot's coherence engine
```

Comment:
Perhaps add a note about rebuilding the microcode when the CCE ucode files get updated. We could also omit this since we will eventually add the "ucode on demand" feature to building.

Reply:
Added